source/_templates/installations/manager/configure_indexer_connection.rst

```diff
@@ -1,12 +1,14 @@
 .. Copyright (C) 2015, Wazuh, Inc.
 
+By default, the host is set to localhost: ``<host>https://0.0.0.0:9200/host>``. Replace it with your Wazuh indexer address accordingly. You can use either IP addresses or hostnames.
```

Comment:
Typo: `9200/host>`
Missing `<`

---

I think that we should keep the 0.0.0.0 IP if it is the default value but mention the change, something like:

> Change 0.0.0.0 by your Wazuh indexer node IP address or hostname.

But only for this section, the following code blocks can maintain the custom IPs as they seem to be an example, what do you think?

Reply:
```suggestion
By default, the host is set to localhost: ``<host>https://0.0.0.0:9200</host>``. Replace it with your Wazuh indexer address accordingly. You can use either IP addresses or hostnames.
```